cli/mender-artifact/dump_test.go

```diff
@@ -68,22 +68,38 @@ func runAndCollectStdout(args []string) (string, error) {
 		return "", goRoutineErr
 	}
 
+	// Trim nuls (from --print0-cmdline).
+	if printed[len(printed)-1] == 0 {
+		printed = printed[:len(printed)-1]
+	}
 	return strings.TrimSpace(string(printed)), nil
 }
 
 func TestDumpContent(t *testing.T) {
-	for _, imageType := range []string{"rootfs-image", "my-own-type"} {
-		t.Run(imageType, func(t *testing.T) {
-			testDumpContent(t, imageType)
-		})
+	for _, printCmdline := range []string{"--print-cmdline", "--print0-cmdline"} {
+		for _, imageType := range []string{"rootfs-image", "my-own-type"} {
+			t.Run(fmt.Sprintf("%s/%s", imageType, printCmdline), func(t *testing.T) {
+				testDumpContent(t, imageType, printCmdline)
```

Comment:
```suggestion
			       t.Parallel()	
				testDumpContent(t, imageType, printCmdline)
```

Reply:
Good idea, it should work well here (in theory!).

Reply:
This unfortunately didn't work because the addition of `Parallel` causes Golang to print extra output, which somehow gets picked up by our stdout hook. The test runs pretty quickly even without it, so I'm just gonna leave it as is.

Reply:
in theory...